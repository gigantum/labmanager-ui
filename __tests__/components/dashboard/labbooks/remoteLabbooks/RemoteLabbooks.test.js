import React from 'react'
import renderer from 'react-test-renderer';
import {shallow, mount} from 'enzyme';
import history from 'JS/history'

import json from './__relaydata__/RemoteLabbooks.json'
import RemoteLabbooks from 'Components/dashboard/labbooks/remoteLabbooks/RemoteLabbooks';
import relayTestingUtils from 'relay-testing-utils'

history.location.pathname = 'hostname/labbooks/cloud'

const variables = {first:20}

const fixtures = {
  remoteLabbooks: json.data.labbookList,
  labbookList: json.data.labbookList,
  labbookListId: json.data.labbookList.remoteLabbooks.id,
  auth: {
    login: ()=>{
    }
  },
  history: history,
  showModal: ()=>{},
  goToLabbook: ()=>{},
  filterLabbooks: (labbooks, filter)=>{
    return labbooks
  },
  filterState: 'cloud',
  setFilterValue: ()=>{},
  forceLocalView: ()=>{},
  changeRefetchState: ()=>{},

}



test('Test RemoteLabbooks rendering', () => {

  const localLabbooks = renderer.create(

     relayTestingUtils.relayWrap(<RemoteLabbooks {...fixtures} />, {}, json.data)

  );

  const tree = localLabbooks.toJSON()


  expect(tree).toMatchSnapshot()
})


// describe('RemoteLabbooks panel', () => {
//
//   it('has length of 22', ()=> {
//     const localLabbooks = shallow(
//
//        <RemoteLabbooks history={history} {...fixtures} feed={json.data}/>
//
//     );
//
//     expect(localLabbooks.find('.RemoteLabbooks__panel')).toHaveLength(22)
//   })
//
//
//
//   it('loads more', () => {
//       const relay = {loadMore: () => {}}
//
//       const localLabbooks = mount(
//
//          <RemoteLabbooks relay={relay} history={history} {...fixtures} feed={json.data}/>
//
//       );
//
//       expect(localLabbooks.find('.RemoteLabbooks__panel')).toHaveLength(22)
//   })
//
//
//   it('show modal', () => {
//     const relay = {loadMore: () => {}}
//     const localLabbooks = mount(
//
//        <RemoteLabbooks relay={relay} history={history} {...fixtures} feed={json.data}/>
//     );
//
//     localLabbooks.find('.RemoteLabbooks__title').simulate('click')
//
//     expect(localLabbooks.node.refs.wizardModal.state.modal_visible).toBeTruthy()
//
//   })
//
//
//   it('show modal by panel', () => {
//     const relay = {loadMore: () => {}}
//     const localLabbooks = mount(
//
//        <RemoteLabbooks relay={relay} history={history} {...fixtures} feed={json.data}/>
//     );
//     localLabbooks.find('.RemoteLabbooks__panel--add').simulate('click')
//
//     expect(localLabbooks.node.refs.wizardModal.state.modal_visible).toBeTruthy()
//
//   })
// })
//
//
//
// describe('Test RemoteLabbooks click', () => {
//
//   const localLabbooks = mount(
//
//       <RemoteLabbooks history={history} feed={json.data}/>
//
//   );
//
//   localLabbooks.at(0).simulate('click')
//   localLabbooks.setState({'labbookName': json.data.localLabbooks.edges[0].node.name})
//
//   expect(localLabbooks.node.state.labbookName === json.data.localLabbooks.edges[0].node.name).toBeTruthy();
// })
//
//
// describe('Test RemoteLabbooks edges output', () => {
//
//   const localLabbooks = mount(
//
//       <RemoteLabbooks history={history} feed={json.data}/>
//
//   );
//
//   expect(localLabbooks.find('.RemoteLabbooks__labbooks .RemoteLabbooks__text-row h4').at(0).text()).toEqual(json.data.localLabbooks.edges[0].node.name)
//
// })
//
// describe('Test scroll functon', () => {
//
//   const localLabbooks = mount(
//
//       <RemoteLabbooks history={history} feed={json.data}/>
//
//   );
//
//   localLabbooks.find('.RemoteLabbooks__panel').at(20).simulate('scroll')
//
//
//
//   expect(localLabbooks.find('.RemoteLabbooks__panel')).toHaveLength(22)
//
// })




export default variables
