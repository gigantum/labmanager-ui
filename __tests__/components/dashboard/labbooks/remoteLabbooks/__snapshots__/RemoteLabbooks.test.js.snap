// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test RemoteLabbooks rendering 1`] = `
<div
  className="Labbooks__listing"
>
  <div
    className="grid"
  >
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                update-test
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by dmk
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                trump-tweets
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by dmk
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                tpub
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by bill2
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                test-lb
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by dmk
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                testing-notifications
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by dmk
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                testing-activity-feed
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                test-book
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                temp-labook22
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                temp
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                t3
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by bill2
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                spade
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                snappy
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by gigantum-examples
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                set-public-test-calum-true
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                set-public-test-calum
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                sdfdsfsdfsdf
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by joel
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                rootpy3
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by gigantum-examples
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                rootpy3
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by randalg
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                random-picture-generator
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud-download"
          disabled={false}
          onClick={[Function]}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete-disabled"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                python-data-science-handbook
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by dmk
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="Card column-4-span-3 flex flex--column justify--space-between"
    >
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--icon"
      >
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--cloud"
          disabled={true}
        />
        <button
          className="RemoteLabbooks__icon RemoteLabbooks__icon--delete"
          disabled={false}
          onClick={[Function]}
        />
      </div>
      <div
        className="RemoteLabbooks__row RemoteLabbooks__row--text"
      >
        <div
          className="RemoteLabbooks__row RemoteLabbooks__row--title"
        >
          <h6
            className="RemoteLabbooks__panel-title"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                pulish-temp4
              </span>
            </span>
          </h6>
        </div>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--owner"
        >
          Created by cbutler
        </p>
        <p
          className="RemoteLabbooks__paragraph RemoteLabbooks__paragraph--description"
        >
          <span
            className={undefined}
          />
        </p>
      </div>
      <div
        className="Tooltip RemoteLabbooks__undefined"
        data-tooltip="undefined"
      />
    </div>
    <div
      className="LabbookPaginationLoader--hidden"
    >
      <div />
    </div>
    <div
      className="LabbookPaginationLoader--hidden"
    >
      <div />
    </div>
    <div
      className="LabbookPaginationLoader--hidden"
    >
      <div />
    </div>
    <div
      className="LabbookPaginationLoader--hidden"
    >
      <div />
    </div>
    <div
      className="LabbookPaginationLoader--hidden"
    >
      <div />
    </div>
  </div>
</div>
`;
